src/coord/src/coord.rs

```diff
@@ -611,30 +581,29 @@ impl Coordinator {
                         }])
                         .await
                         .unwrap();
+                    self.initialize_storage_read_policies(
+                        vec![entry.id()],
+                        self.logical_compaction_window_ms,
+                    )
+                    .await;
                 }
                 CatalogItem::Index(_) => {
                     if BUILTINS.logs().any(|log| log.index_id == entry.id()) {
-                        // Indexes on logging views are special, as they are
-                        // already installed in the dataflow plane via
-                        // `dataflow_types::client::Command::EnableLogging`. Just teach the
-                        // coordinator of their existence, without creating a
-                        // dataflow for the index.
-                        //
-                        // TODO(benesch): why is this hardcoded to 1000?
```

Comment:
Revive this TODO comment, as it is still relevant.

Reply:
Done.